SampleObjC/sdk-ios-objc/OCViewController.m

```diff
@@ -0,0 +1,109 @@
+//
+//  ViewController.m
+//  sdk-ios-objc
+//
+//  Created by Simon Hamadene on 16/10/2018.
+//  Copyright © 2018 Yoti Limited. All rights reserved.
+//
+
+#import "OCViewController.h"
+#import <YotiButtonSDK/YotiButtonSDK.h>
+#import "OCProfileViewController.h"
+
+@interface OCViewController () <YTBSDKDelegate, YTBBackendDelegate>
+@property (weak, nonatomic) IBOutlet YotiButton *rememberMe;
+@property (weak, nonatomic) IBOutlet YotiButton *selfieAuth;
+@end
+
+@implementation OCViewController {
+    NSArray *responseObject;
+}
```

Comment:
Isn't here has an extra `}` ?

Reply:
there is no an extra `}` . I don't think bit would build otherwise. :-)